lib/views/frontend/spree/checkout/payment/_stripe.html.erb

```diff
@@ -41,7 +41,7 @@
       Spree.stripePaymentMethod.append("<input type='hidden' class='stripeToken' name='payment_source[" + paymentMethodId + "][last_digits]' value='" + response.card.last4 + "'/>");
       Spree.stripePaymentMethod.append("<input type='hidden' class='stripeToken' name='payment_source[" + paymentMethodId + "][month]' value='" + response.card.exp_month + "'/>");
       Spree.stripePaymentMethod.append("<input type='hidden' class='stripeToken' name='payment_source[" + paymentMethodId + "][year]' value='" + response.card.exp_year + "'/>");
-      return Spree.stripePaymentMethod.parents("form").get(0).submit();
+      return Spree.stripePaymentMethod.parents("form").trigger('submit');
```

Comment:
I believe the get(0) still needs to be there in case multiple payment methods are setup.